DevelopGitHub/GitHubExtension/GitHubExtention.Preferences.WebApi/Azure/AzureBlobStorageMultipartProvider .cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using GitHubExtention.Preferences.WebApi.Queries;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace GitHubExtention.Preferences.WebApi
+{
+    public class AzureBlobStorageMultipartProvider : MultipartFileStreamProvider
```

Comment:
Please hide all details of implementation by interface. SOLID...Class should be acceptable for extension not for MODIFICATION! The second part you should not rely on implementation of MultipartFileStreamProvider it can be changed...